### PR TITLE
Correction in labels in -Pnew configuration  

### PR DESCRIPTION
**Pull request by theexplorist**

### CODE REVIEW

**Comment by dev-gaur** [comment]
fixes #1678

File: samples/xml-config/pom.xml [hunk 1]
```diff
@@ -241,7 +243,7 @@
 		  <name>${image.user}/${project.artifactId}:${project.version}</name>
 		  <alias>camel-service</alias>
 		  <build>
-		    <from>fabric8/s2i-java@@sha256:a6ba8a6e76ebd75f99f2431fe0ec39a82985bc21c0f9bd56021deca1ffde520c</from>
+		    <from>fabric8/s2i-java</from>
```
Comment (rohanKanojia) [inline samples/xml-config/pom.xml]: Just need to remove one `@` here,

File: samples/xml-config/pom.xml [hunk 2]
```diff
@@ -211,6 +212,7 @@
                   <volumes>
                   </volumes>
                 </deployment>
+                </labels>
```
Comment (rohanKanojia) [inline samples/xml-config/pom.xml]: wrong place to close label

File: samples/xml-config/pom.xml [hunk 3]
```diff
@@ -182,8 +182,9 @@
               <resources>
                 <!-- Labels that are applied to all created objects -->
                 <labels>
-                  <group>quickstarts</group>
-                </labels>
+                    <all>
+                        <group>quickstarts</group>
+                    </all>
```
Comment (rohanKanojia) [inline samples/xml-config/pom.xml]: add `</label>` to close labels field here

Comment (theexplorist) [reply samples/xml-config/pom.xml]: deployments is one of the labels in metadataconfig. On the contrary there is no field deployment in ResourceConfig hence there build fails. And I have tested the build, now it passes.

Comment (rohanKanojia) [reply samples/xml-config/pom.xml]: oh, okay. Have you fixed this in external profile as well?

File: samples/xml-config/pom.xml (hunk 1 above)
Comment (theexplorist) [reply samples/xml-config/pom.xml]: Oh there are 2 @s I had not took notice!

File: samples/xml-config/pom.xml (hunk 3 above)
Comment (theexplorist) [reply samples/xml-config/pom.xml]: Yes everything is correct there, adding the digest again!

Comment (theexplorist) [reply samples/xml-config/pom.xml]: Fixed, now i guess it is good to merge @rohanKanojia

**Pull request merged**